telemetry/telemetry.go

```diff
@@ -255,10 +255,10 @@ func APIError(method string, path string, e error) {
 	)
 }
 
-// Report a failure withoout a specific error object
+// Report a failure without a specific error object
 func Failure(message string) {
 	analyticsClient.Track(distinctID(),
-		message,
+		fmt.Sprintf("Unknown Error: %s", message),
```

Comment:
I am unsure about the need to add the prefixes as you have done here.  I don't think we really felt their absence using Mixpanel, and the "type" field could be used to filter.  Is this an Amplitude or Postman best practice that we should understand?

We should not use "Unknown Error" here, though, because the failures can be known errors that simply don't come with a Go "error".

Reply:
I have just added these prefixes to follow a convention with the event names, also it can help us in the easier resolution of new events that might be added in the Amplitude.
Amplitude doesn't automatically start tracking a new event, we need to tell Amplitude to start tracking the event. Like in the attached image, events that are not being tracked are marked as Unexpected we need to add them to the list to make them live. 
https://app.amplitude.com/data/postman/Postman/events/POA-41-migrate-to-amplitude/latest?view=All
<img width="1219" alt="image" src="https://github.com/akitasoftware/akita-cli/assets/116863733/35dabbf1-5524-4b6c-b4fe-2c72b9ac0a34">